Buttplug.Core/ButtplugDevice.cs

```diff
@@ -21,30 +23,89 @@ public bool IsConnected
             }
         }
 
+        public uint VibratorCount

```

Comment:
Shouldn't this be FeatureCount now?

Reply:
Urrg. This was from the first cut where I didn't want to define an array of vibrator speeds in every class. But it only applies to vibrators, because they're simple enough to fit; rotators have a direction and linear have position in addition to speeds.

I'm just going to refactor this out.